From 3b9036aab355a4600f57475b84ded8fc30b50093 Mon Sep 17 00:00:00 2001
From: Greg Hackmann <ghackmann@google.com>
Date: Wed, 16 Oct 2013 14:53:12 -0700
Subject: [PATCH 289/634] video: adf: set default interface dpms_state to OFF

Interfaces default to unplugged, so they should also default to off

Change-Id: I36500a54b11f354a0d7dd2c9924a79e0d9c6f855
Signed-off-by: Greg Hackmann <ghackmann@google.com>
---
 drivers/video/adf/adf.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/adf/adf.c b/drivers/video/adf/adf.c
index 64f763d..0ae51ac 100644
--- a/drivers/video/adf/adf.c
+++ b/drivers/video/adf/adf.c
@@ -676,6 +676,7 @@ int adf_interface_init(struct adf_interface *intf, struct adf_device *dev,
 	intf->idx = idx;
 	intf->flags = flags;
 	intf->ops = ops;
+	intf->dpms_state = DRM_MODE_DPMS_OFF;
 	init_waitqueue_head(&intf->vsync_wait);
 	rwlock_init(&intf->vsync_lock);
 	rwlock_init(&intf->hotplug_modelist_lock);
-- 
1.9.1

